tla/consensus/ccfraft.tla

```diff
@@ -1143,6 +1152,7 @@ RcvProposeVoteRequest(i, j) ==
         /\ Timeout(m.dest)
         /\ Discard(m)
 
+\* Node i receives a message from node j.
```

Comment:
I think `i` and `j` are poor choices, and we ought to move to `src`, `dst` eventually.

Reply:
+1